### ISSUE TITLE
开启虚拟表格导致固定在右边的操作列不显示了

### ISSUE THREAD

**Issue by boycot2015** [u-leo/umy-ui#5]
开启虚拟表格导致固定在右边的操作列不显示了
![image](https://user-images.githubusercontent.com/34299341/93191196-d5007500-f776-11ea-8bf6-e914aa6107fc.png)

**Comment by livelyPeng** [comment]
样式问题 你可能需要加群 给你说了

**Comment by CalmHarbin** [comment]
> 开启虚拟表格导致固定在右边的操作列不显示了
> ![image](https://user-images.githubusercontent.com/34299341/93191196-d5007500-f776-11ea-8bf6-e914aa6107fc.png)

我是没有引入样式导致的
 import 'umy-ui/lib/theme-chalk/index.css';// 引入样式

**Comment by whatlin** [comment]
> 

这个问题有没有解决

**Comment by CalmHarbin** [comment]
> 

我是没有引入样式导致的
import 'umy-ui/lib/theme-chalk/index.css';// 引入样式

**Comment by whatlin** [comment]
> > 
> 
> 我是没有引入样式导入的 import 'umy-ui/lib/theme-chalk/index.css';// 引入样式

要重新引入他的umy-ui样式的吗？

**Comment by anqila89757** [comment]
开启虚拟表格，用的ux-grid, <ux-table-column width="55" title="序号" type="index" fixed /> 序号固定列不生效，固定不了

**Comment by web66688888** [comment]
虚拟列表  刷新列表滚动条会被重置，不能恢复到原来滚动的位置

**Comment by crow04** [comment]
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by sfcollins-v8m** [comment]
@crow04 I agree with the points raised in this issue. When both `use-virtual` and `fixed="right"` are employed simultaneously, the sticky column fails to display the intended content. Removing `use-virtual` from the table parameters resolves the issue. Please revisit this issue at your earliest convenience. Thank you

**Comment by livelyPeng** [comment]
@sfcollins-v8m as mentioned above, you need to import a style file
